python/lsst/base/packages.py

```diff
@@ -159,8 +162,7 @@ def getEnvironmentPackages():
         from eups import Eups
         from eups.Product import Product
     except ImportError:
-        from lsst.pex.logging import getDefaultLog
-        getDefaultLog().warn("Unable to import eups, so cannot determine package versions from environment")
+        log.warning("Unable to import eups, so cannot determine package versions from environment")
```

Comment:
Have you tested this? I thought it was `log.warn`.

Reply:
Yes. It works (we noticed this because eups did not load for @gcomoretto). Python `logging` prefers `logging.warning` and deprecates `logging.warn` (we don't use `lsst.log` here)